assets/tests/components/mapPage.test.tsx

```diff
@@ -667,6 +675,47 @@ describe("<MapPage />", () => {
     )
   })
 
+  test("When a location is selected from the list of search results, location card should be visible", async () => {
+    jest.spyOn(global, "scrollTo").mockImplementationOnce(jest.fn())
+
+    const location = locationSearchResultFactory.build()
+
+    ;(useLocationSearchResults as jest.Mock).mockReturnValue([location])
+
+    const { container } = render(
+      <StateDispatchProvider
+        state={stateFactory.build({
+          searchPageState: activeSearchPageStateFactory.build({
+            query: { text: location.name! },
+            selectedEntity: {
+              type: SelectedEntityType.Location,
+              location: location,
+            },
+          }),
+        })}
+        dispatch={jest.fn()}
+      >
+        <MapPage />
+      </StateDispatchProvider>
+    )
+
+    const mapSearchPanel = getMapSearchPanel()
+    expect(mapSearchPanel).toHaveClass("c-map-page__input-and-results--visible")
+
+    const locationCard = screen.getByLabelText(location.name!)
```

Comment:
thought (non-blocking): maybe it would be worth defining a named region for the selected entity that could be queried `within` to help with asserting that the found card is selected and not a result. That said, I think this is test is OK as-is!

Reply:
Yeah that's sort of what I was trying to do (indirectly) with the test that the Street View button is visible. I'm actually going to add an assertion that the "New Search" button is there though as a way of making sure that this is a selected result.